hooks/pre-commit

```diff
@@ -8,12 +8,22 @@ KUBE_HOOKS_DIR="$(dirname "$(test -L "$0" && echo "$(dirname $0)/$(readlink "$0"
 
 exit_code=0
 
+echo -ne "Checking that it builds... "
+if ! OUT=$("hack/build-go.sh" 2>&1); then
```

Comment:
Not sure how I feel about the pre-commit hook blowing away the binaries in my directory.